Mathlib/MeasureTheory/Measure/Hausdorff.lean

```diff
@@ -1093,6 +1095,23 @@ theorem hausdorffMeasure_segment {E : Type*} [NormedAddCommGroup E] [NormedSpace
     [MeasurableSpace E] [BorelSpace E] (x y : E) : μH[1] (segment ℝ x y) = edist x y := by
   rw [← affineSegment_eq_segment, hausdorffMeasure_affineSegment]
 
+/--
+Let `s` be a subset of `𝕜`-inner product space, and `K` a subspace. Then the `d`-dimensional
+Hausdorff measure of the orthogonal projection of `s` onto `K` is less than or equal to the
+`d`-dimensional Hausdorff measure of `s`.
+-/
+theorem hausdorffMeasure_orthogonalProjection_le {𝕜 : Type u_1} {E : Type u_2} [RCLike 𝕜]
+  [NormedAddCommGroup E] [InnerProductSpace 𝕜 E] [MeasurableSpace E] [BorelSpace E]
+  (K : Submodule 𝕜 E) [K.HasOrthogonalProjection]
+  (d : ℝ) (s : Set E) (hs : 0 ≤ d) :
+    μH[d] (K.orthogonalProjection '' s) ≤ μH[d] s := by
+  have h₁ : LipschitzWith 1 (K.orthogonalProjection) := by
+    apply Submodule.norm_orthogonalProjection_apply
+  have h₂ : μH[d] (Submodule.orthogonalProjection K '' s) ≤ 1 ^ d * μH[d] s := by
+    apply LipschitzWith.hausdorffMeasure_image_le
+    exact h₁; exact hs
```

Comment:
I expect this can be one-line somehow - what's the suggestion if you do `by?` instead of `by`